daml-assistant/daml-helper/src/DA/Daml/Helper/Ledger.hs

```diff
@@ -323,6 +327,24 @@ downloadPackage args pid = do
     convPid :: LF.PackageId -> L.PackageId
     convPid (LF.PackageId text) = L.PackageId $ TL.fromStrict text
 
+data RemoteDalf = RemoteDalf
+    { remoteDalfFp :: FilePath
```

Comment:
I think it might be helpful to add a comment that this isn’t the filepath we’ve written to, it’s the file path we’re planning to write it to. Alternatively calli t something like `remoteDalfName`, make it a string and add the extension at the callsite.

Reply:
renamed to `remoteDalfName`.